src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -1,20 +1,24 @@
 package bhs.devilbotz.subsystems;
 
+import bhs.devilbotz.Constants.DriveConstants;
 import com.ctre.phoenix.motorcontrol.InvertType;
 import com.ctre.phoenix.motorcontrol.NeutralMode;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 import com.kauailabs.navx.frc.AHRS;
-import edu.wpi.first.math.filter.SlewRateLimiter;
 import edu.wpi.first.wpilibj.SPI;
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class DriveTrain extends SubsystemBase {
   // Define talons
-  private static final WPI_TalonSRX leftMaster = new WPI_TalonSRX(3);
-  private static final WPI_TalonSRX rightMaster = new WPI_TalonSRX(4);
-  private static final WPI_TalonSRX leftFollower = new WPI_TalonSRX(1);
-  private static final WPI_TalonSRX rightFollower = new WPI_TalonSRX(2);
+  private static final WPI_TalonSRX leftMaster =
+      new WPI_TalonSRX(DriveConstants.MOTOR_LEFT_MASTER_CAN_ID);
```

Comment:
since the class is static, you can directly reference the constants class instead of importing it, to avoid an extra import.
Ex:
Constants.DriveConstants.MOTOR_LEFT_MASTER_CAN_ID

Reply:
Interestingly enough, it didn't recognize the constants until I added the import.  We would at least need: `import bhs.devilbotz.Constants`?